Formula/z/zim.rb

```diff
@@ -12,6 +12,7 @@ class Zim < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python-setuptools" => :build
```

Comment:
Can we get rid of `python-setuptools`?

Reply:
done, also switched to a venv since that is a more typical installation

Reply:
Thanks!